data/README.md

```diff
@@ -0,0 +1,9 @@
+# Language-Specific Data
+
+This directory contains language-specific data files. Most importantly, you will find here:
+
+1. a list of unique characters for the target language (e.g. English) in `data/alphabet.txt`
+2. a binary n-gram language model compiled by `kenlm` in `data/lm/lm.binary`
+3. a trie model compiled by `kenlm` in `data/lm/trie`
```

Comment:
The trie file is not created by KenLM, it's created by our ctcdecode code.

Reply:
And maybe capitalize the items?

Reply:
fixed in two new commits